Dockerfile-slim.template

```diff
@@ -18,9 +18,9 @@ RUN set -ex; \
 # uid                  PostgreSQL Debian Repository
 	key='B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8'; \
 	export GNUPGHOME="$(mktemp -d)"; \
-	(gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" \
-	    || gpg --keyserver ipv4.pool.sks-keyservers.net --recv-keys "$key"); \
-	gpg --export "$key" > /etc/apt/trusted.gpg.d/postgres.gpg; \
+	(gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" \
+	    || gpg --batch --keyserver ipv4.pool.sks-keyservers.net --recv-keys "$key"); \
```

Comment:
Clean up from #64, we should apply this diff, as we're not going to use the original solution from #56:

```diff
diff --git a/Dockerfile-slim.template b/Dockerfile-slim.template
index 82b0ae9..423b752 100644
--- a/Dockerfile-slim.template
+++ b/Dockerfile-slim.template
@@ -18,8 +18,7 @@ RUN set -ex; \
 # uid                  PostgreSQL Debian Repository
        key='B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8'; \
        export GNUPGHOME="$(mktemp -d)"; \
-       (gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" \
-           || gpg --keyserver ipv4.pool.sks-keyservers.net --recv-keys "$key"); \
+       gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
        gpg --export "$key" > /etc/apt/trusted.gpg.d/postgres.gpg; \
        rm -rf "$GNUPGHOME"; \
        apt-key list
```